examples/eSealing/Dockerfile

```diff
@@ -5,12 +5,14 @@ FROM ${base_image} AS build
 WORKDIR /app
 COPY ./package.json ./package.json
 COPY ./tsconfig.json ./tsconfig.json
-COPY ./esbuild.config.ts ./
-COPY ./types ./types
+COPY ./index.html ./
+COPY ./vite.config.ts ./
+COPY ./public ./public
 COPY ./src ./src
 
 RUN yarn && yarn cache clean
 RUN yarn build
 
 FROM nginx
 COPY --from=build ./app/dist ./usr/share/nginx/html
+COPY ./nginx.conf /etc/nginx/conf.d/default.conf
```

Comment:
What do you mean? I updated the e-sealing project to the new SDK. Do you mean the nginx conf in particular? It suffered from the same case of very large files as wCCD, so I did the same overhaul here.

Reply:
Yes, I mean that particular line. I think it would have been better to not do other things than _just_ upgrading the web-sdk, as that is already a large change, to make it easier to review. So my question was mostly if you had to do it here, but I assume not.